Plugins/Public/alley/PlayerRestrictions.cpp

```diff
@@ -675,6 +675,19 @@ bool UserCmd_JettisonAll(uint iClientID, const wstring &wscCmd, const wstring &w
 			pub::IsCommodity(item->iArchID, flag);
 			if (!item->bMounted && flag)
 			{
+				bool skipItem = false;
+				for (map<uint, string>::iterator i = notradelist.begin(); i != notradelist.end(); ++i)
+				{
+					if (i->first == item->iArchID)
+					{
+						PrintUserCmdText(iClientID, L"ERR you can't jettison %s.", stows(i->second).c_str());
```

Comment:
I'm not sure this is technically something we should respond with ERR to. Just silently don't drop it if the player is not allowed?